sjb/config/test_cases/test_branch_online_console_extensions.yml

```diff
@@ -0,0 +1,13 @@
+---
+parent: 'common/test_cases/origin.yml'
+overrides:
+  sync_repos:
+    - name: "online-console-extensions"
+  junit_analysis: False
+extensions:
+  actions:
+    - type: "script"
+      title: "online-console-extensions generated template check"
+      repository: "online-console-extensions"
+      script: |-
+        ./hack/check-generated-templates.sh
```

Comment:
@stevekuznetsov that's all i need (until we add more checks/tests to the console-extensions repo)?

Reply:
SGTM -- have you pushed the job out and tested it?

Reply:
NVM, need the new AMI to do the clone correctly. duh.